coq-sail-riscv.opam

```
@@ -2,34 +2,24 @@ opam-version: "2.0"
 name: "coq-sail-riscv"
 version: "0.5"
 maintainer: "Sail Devs <cl-sail-dev@lists.cam.ac.uk>"
-authors: [
-  "Alasdair Armstrong"
-  "Thomas Bauereiss"
-  "Brian Campbell"
-  "Shaked Flur"
-  "Jonathan French"
-  "Prashanth Mundkur"
-  "Robert Norton"
-  "Christopher Pulte"
-  "Peter Sewell"
-]
-homepage: "https://github.com/rems-project/sail-riscv/"
-bug-reports: "https://github.com/rems-project/sail-riscv/issues"
-license: "BSD3"
-dev-repo: "git+https://github.com/rems-project/sail-riscv.git"
+authors: "Sail RISC-V contributors"
+homepage: "https://github.com/riscv/sail-riscv/"
+bug-reports: "https://github.com/riscv/sail-riscv/issues"
+license: "BSD-2-Clause"
+dev-repo: "git+https://github.com/riscv/sail-riscv.git"
 build: [
-  [make "ARCH=RV64" "riscv_coq_build"]
-  [make "ARCH=RV32" "riscv_coq_build"]
+  ["cmake" "-S" "." "-B" "build" "-DCMAKE_BUILD_TYPE=RelWithDebInfo" "-DDOWNLOAD_GMP=FALSE"]
+  ["cmake" "--build" "build" "--target" "build_rocq_rv64d"]
+  ["cmake" "--build" "build" "--target" "build_rocq_rv32d"]
```

You can pass multiple targets here separated by spaces apparently...

```suggestion
  ["cmake" "--build" "build" "--target" "build_rocq_rv32d" "build_rocq_rv64d"]
```